### ISSUE TITLE
class 'app-portal' disappeared from <body>

### ISSUE THREAD

**Issue by hdijkema** [espocrm/espocrm#3143]
**Describe the bug**

Although in  `./application/Espo/Core/Portal/ApplicationRunners/Client.php`  there's still the attribute:

```php
 'appClientClassName' => 'app-portal',
```

It is not used anymore in the `<body>` element of EspoCRM. 

I noticed this as from version 8.0.6 of EspoCRM, while upgrading from version 6.1.8. 

I depend on this class in my custom css file to be able to customize the portal look and feel for my client. 

**To Reproduce (MANDATORY, DO NOT REMOVE)**
Explicit steps to reproduce the behavior:
1. Run version 6.1.8 of EspoCRM. 
2. In the normal CRM mode, `<body>`  will contain class `classname="app"`. In the portal mode, `<body>` will contain `classname="app-portal"`. 
3. Run version 8.0.6 of EspoCRM. 
4. Both `classname="app"` and `classname="app-portal"` have disappeared and are replaced by `classname="has-navbar"`.

**Expected behavior**
I would expect something like 

```html
<body classname="app has-navbar" ...>
```
for the normal CRM, and

```html
<body classname="app-portal has-navbar" ...>
```

for the portal CRM. 

**Screenshots**
not applicable. 

**EspoCRM version**
I noticed this in version 8.0.6. I'm still upgrading to the following versions, until 8.3.x, so I currently don't know if this has been noticed and solved. 

**Additional context**
-

**Comment by yurikuzn** [comment]
I don't remember we ever had *appClientClassName* added to an element class list. It refers to a JS class (ID), not CSS.

**Comment by hdijkema** [comment]
I stand corrected. My apologies. I made a customization of html/main.html indeed. Added a class there. It's never good to do put customizations directly in the source of EspoCRM. 

Still a question though. In commit [d7b9b65](https://github.com/espocrm/espocrm/commit/d7b9b65990cd3a99ef3e28f879d1abbeef2d025f) there's an application id in the body tag. 

![image](https://github.com/user-attachments/assets/feea0828-41c6-46a3-abf1-692902ff0965)

I recon this is for version 8.4 of EspoCRM?
